### DISCUSSION TITLE
Add information about currently selected video track

### DISCUSSION THREAD

**Discussion by maciejsiiminski** [TheWidlarzGroup/react-native-video#4954] [Ideas]
### What problem are you trying to solve?

Currently, I cannot get information about the selected video track. Video tracks returned in the onLoad data do not have the selected property. OnVideoTracks event contains this information but it is fired only when the tracks change so it is impossible to use it in order to get information about the initially selected video track.

This info is needed in the app I developed for diagnostics.

### Proposed solution

I propose either adding the selected property to the video track info on load or firing onVideoTracks when the initial active video track is selected.

**Comment by freeboub** [comment]
It makes sense, but we need to keep in mind that ABR can lead to multiple selected tracks and a way to reselect multiple tracks then!

**Comment by maciejsiiminski** [comment]
Oh, it is good to know because ABR is used in my app. So in the case of ABR, all possible tracks are selected? If so it would be nice to also have information on which one is actually active.

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
